step-capstone/src/__tests__/CountCatTests.test.js

```diff
@@ -0,0 +1,67 @@
+import '@testing-library/jest-dom'
+import {countCat} from '../scripts/Suggestion'
```

Comment:
+1 on this. It would be ideal if we can make the mappings deterministic to avoid a source of flakiness as a best practice thing but I agree that it would be a nice-to-have instead of need-to-have for the purpose of the capstone.

Reply:
I agree. Added to our (growing) list of nice to have TODOS.

Reply:
Good points. I think we can implement these ideas on week 11.